data/archived_mur_citation_map.csv

```diff
@@ -0,0 +1,252 @@
+2:431,52:30101
```

Comment:
I think using a data file is the preferred way, to keep the data separate from the code. It's also the approach we use for other data loading tasks, like the [archive MUR names](https://github.com/18F/openFEC/blob/develop/webservices/load_legal_docs.py#L382).

Hmm, not sure how I didn't notice the duplicates, but I'll remove them and rename the file.